Dizzy/Screens/Login/LoginVC.swift

```diff
@@ -153,7 +153,7 @@ final class LoginVC: UIViewController, LoadingContainer, PopupPresenter {
     }
     
     private func setupNavigationView() {
-        self.navigationItem.title = "Login".localized
+        self.navigationItem.title = "".localized
```

Comment:
please remove this line, if we dont want any title we should not set it

Reply:
done